src/app/components/normal-component/chat/chat.component.ts

```diff
@@ -38,21 +39,21 @@ export class ChatComponent implements OnInit, AfterViewChecked, OnDestroy {
     @ViewChild('chatDiv') chatDiv: ElementRef<HTMLElement>;
 
     constructor(private readonly chatService: ChatService,
-                private readonly authenticationService: AuthenticationService) {
+                private readonly connectedUserService: ConnectedUserService) {
         display(ChatComponent.VERBOSE, 'ChatComponent constructor');
     }
     public ngOnInit(): void {
         display(ChatComponent.VERBOSE, `ChatComponent.ngOnInit for chat ${this.chatId}`);
 
         assert(this.chatId != null && this.chatId !== '', 'No chat to join mentionned');
-        this.authSubscription = this.authenticationService.getUserObs()
+        this.authSubscription = this.connectedUserService.getUserObs()
             .subscribe((user: AuthUser) => {
-                if (this.isConnectedUser(user)) {
+                if (this.userJustConnected(user)) {
                     display(ChatComponent.VERBOSE, JSON.stringify(user) + ' just connected');
                     this.username = user.username;
                     this.connected = true;
```

Comment:
tricky tricky: si l'utilisateur change de nom, ça sera plus mis à jour !

Reply:
mais dans ma branche y'a plus tout ce bouzin d'auth géré dans le chat, ça simplifie beaucoup

Reply:
ah bah je viens de régler ça (to be pushed) pour qu'on update le name dans les update "online -> online" :D

Conflit incoming